app/controllers/CharitableEligibilityController.scala

```diff
@@ -17,32 +17,47 @@
 package controllers
 
 import config.AppConfig
+import controllers.actions.DataRetrievalAction
+import controllers.auth.SessionIdentifierAction
+import controllers.connectors.DataShortCacheConnector
 import forms.EligibilityForm
 import javax.inject.Inject
+import models.YesNoModel
+import play.api.i18n.I18nSupport
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
 import uk.gov.hmrc.play.bootstrap.controller.FrontendController
 import views.html.home.eligibility
 
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
We can pull an execution context from the MessagesControllerComponents rather than importing this one.